CI-Files/test_singularity_gpu.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# install gemmforge
+pip3 install git+https://github.com/ravil-mobile/gemmforge.git
+
+# prepare
+cd ./tests/convergence_elastic && ./generateCubes.sh && cd ../..
+
+for precision in double single; do
+    mkdir -p ./SeisSol/_build_${precision} && cd ./SeisSol/_build_${precision}
+    cp ../../tests/convergence_elastic/* .
+
+    # compile
+    cmake .. -DCMAKE_BUILD_TYPE=Release -DDEVICE_ARCH=${GPU_VENDOR} -DDEVICE_SUB_ARCH=${GPU_MODEL}  -DHOST_ARCH=${HOST} -DPRECISION=${precision}
+    make -j
```

Comment:
This only compiles on 1 thread, is that your intention? The -j without argument confuses me.